Added a scope that includes children by default when loading the comm…

brnlsl opened the pull request:
…ents list.

This can be overridden  (in the CommentAdditions concern) if you’d like to include additional associations in your host app.

brnlsl committed:
Added a scope that includes children by default when loading the comments list.

This can be overridden  (in the CommentAdditions concern) if you’d like to include additional associations in your host app.

bcroesch commented on app/models/flyover_comments/comment.rb:
```diff
@@ -1,6 +1,9 @@
   class Comment < ActiveRecord::Base
     include FlyoverComments::LinkParsing
+
+    scope :with_includes, -> { includes(:children) }
```
I think we probably don't want this by default? Some comment threads (esp. on GJOpen) get really long and we load the children (other than the first one) remotely. I think this would load all of them and just not render most?

brnlsl committed:
Don’t load all the children. 
It has performance risks if there are a lot of children. We only display one child by default, and don’t want to load all of them in case there are many.
